DELibrary.NET/Structs/Quaternion.cs

```diff
@@ -12,11 +12,13 @@ public struct Quaternion
         public float z;
         public float w;
 
+        private static readonly Quaternion _identity = new Quaternion();
```

Comment:
why not replace the public one with this and remove the underscore

Reply:
Fixed.